### PR TITLE
Add more Types needed for existing and next upcoming clusters

### PR DESCRIPTION
**Pull request by Apollon77**
I adjusted two types and added several more needed by my next upcoming PRs :-)

### CODE REVIEW

File: src/matter/common/AttributeId.ts
Comment (mfucci) [inline src/matter/common/AttributeId.ts:18]: Attribute ID

File: src/matter/common/ClusterId.ts
Comment (mfucci) [inline src/matter/common/ClusterId.ts:19]: Cluster Identifier

File: src/matter/common/DeviceTypeId.ts
Comment (mfucci) [inline src/matter/common/DeviceTypeId.ts:18]: Device type ID

File: src/matter/common/EndpointNumber.ts
Comment (mfucci) [inline src/matter/common/EndpointNumber.ts:18]: Endpoint number

File: src/matter/common/FabricId.ts
Comment (mfucci) [inline src/matter/common/FabricId.ts:19]: Fabric ID

File: src/matter/common/FabricId.ts
Comment (mfucci) [inline src/matter/common/FabricId.ts:22]: Fabric ID

File: src/matter/common/FabricId.ts
Comment (mfucci) [inline src/matter/common/FabricId.ts:12]: Missing .

File: src/matter/common/DeviceTypeId.ts
Comment (mfucci) [inline src/matter/common/DeviceTypeId.ts:11]: Missing .

File: src/matter/common/GroupId.ts
Comment (mfucci) [inline src/matter/common/GroupId.ts:26]: convert -> converts

File: src/matter/common/GroupId.ts
Comment (mfucci) [inline src/matter/common/GroupId.ts:29]: Group ID

File: src/matter/common/VendorId.ts
Comment (mfucci) [inline src/matter/common/VendorId.ts:21]: I will add a PR to introduce UInt16T. Maybe you can merge and use it here

**Review by mfucci — APPROVED** [review]
Minor comments about mistakes in the doc.
I will probably refactor those to avoid use the type hack but fine to commit them since it should not change the interface.

File: src/matter/common/AttributeId.ts
Comment (Apollon77) [reply src/matter/common/AttributeId.ts:18]: meeh ... copy paste :-) fixing

File: src/matter/common/VendorId.ts
Comment (Apollon77) [reply src/matter/common/VendorId.ts:21]: Because PR not ready as of your info it is slightly delayed ... :-)

**Comment by Apollon77** [comment]
Sorry for the copy paste errors ... I think my brain was too strange after these hours reading specs ;-)

FIxed ... For the Unbound: As proposed two options. I adjust after you finished your PR and we merged it. Or we adjust your PR that old types are still in and I "convert" - and then we kill the old ones. Would allow a non-blocking working - for me both ok.

I have alreydy type adjustmensts again for all existing CLusters and 9 new ones locally ... :-))

**Comment by mfucci** [comment]
Yes, let's wait until this is merged with the PR with the new types but otherwise everything seems fine for me.

**Comment by mfucci** [comment]
There seems to be a build issue

**Comment by Apollon77** [comment]
Yes, checking it right now

**Comment by Apollon77** [comment]
PS: Sure it won't ...it bases on UIntegerT :-) fixing now

**Comment by Apollon77** [comment]
@mfucci one question: 

```
Bound(UInt16T,{ min: 0, max: 0xFFFF })
```

or can I remove the max here because it is the UInt16T max itself? So is that "merged" or not?

**Comment by Apollon77** [comment]
@mfucci  types adjusted ... builds now

**Review by mfucci — APPROVED** [review]

File: src/matter/common/VendorId.ts
Comment (Apollon77) [reply src/matter/common/VendorId.ts]: @mfucci Do you have an answer here? (also for future work)

**Pull request merged**
